src/components/SearchForm/SearchForm.tsx

```diff
@@ -14,19 +14,30 @@ import { BASE_URL, PATHS } from "../../config/constants"
  */
 const SearchForm = () => {
   const router = useRouter()
+  const [searchTerm, setSearchTerm] = useState(
+    (router?.query?.q as string) || ""
+  )
+  const [searchScope, setSearchScope] = useState("all")
```

Comment:
For this PR, you should also set the select to use `router?.query?.search_scope` or `"all"`. Right now, reading `q` from the URL works but you can do the same for `search_scope`. Unless if we need to wait to implement this?

Reply:
this is the current behavior on the discovery front end. this url https://www.nypl.org/research/research-catalog/search?q=&contributor=spaghetti leads you to an empty search bar with all fields selected. 
when we discussed in refinement, it sounded like Sean wanted to keep everything as close to prod as possible. I don't love a lot the behavior between advanced and regular search, but it starts to get dicey when you consider inputting multiple fields in advanced search or fields that aren't in the regular search select.